pocket-notes/scss/styles.scss

```diff
@@ -0,0 +1,394 @@
+                font-family: "SemiBold";
+                letter-spacing: 2px;
+            }
+             /** Start of view selection **/
+            .view-selection{
+                display: flex;
+                align-items: center;
+
+                p{
+                    color: $text-white;
+                    font-family: "SemiBold";
+                    font-size: 1.2rem;
+                    padding-right: 2rem;
+                }
+                select {
+                    padding: 0.8rem;
+                    width: 260px;
+                    font-size: 1.2rem;
+                    font-family: "Regular";
+                    color: $text-black;
+                    /* inline SVG */
+                    background-image: url("data:image/svg+xml;charset=US-ASCII,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20256%20448%22%20enable-background%3D%22new%200%200%20256%20448%22%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E.arrow%7Bfill%3A%23424242%3B%7D%3C%2Fstyle%3E%3Cpath%20class%3D%22arrow%22%20d%3D%22M255.9%20168c0-4.2-1.6-7.9-4.8-11.2-3.2-3.2-6.9-4.8-11.2-4.8H16c-4.2%200-7.9%201.6-11.2%204.8S0%20163.8%200%20168c0%204.4%201.6%208.2%204.8%2011.4l112%20112c3.1%203.1%206.8%204.6%2011.2%204.6%204.4%200%208.2-1.5%2011.4-4.6l112-112c3-3.2%204.5-7%204.5-11.4z%22%2F%3E%3C%2Fsvg%3E%0A");
+                    background-position: right 10px center;
+                    background-repeat: no-repeat;
+                    background-size: auto 50%;
+                    border-radius:2px;
+                    border:none;
+                    // disable default appearance
+                    outline: none;
+                    -moz-appearance: none;
+                    -webkit-appearance: none;
+                    appearance: none;
+                    &::-ms-expand { display: none };
+                        option {
+                            font-size: 0.5rem;
+                        }
+                    
+                }
+            }
+             /** End of view selection **/
+         }
+          /** End of container **/
+     }
+    /** End of header **/
+
+     /** Start of notes-wrapper **/
+     .notes-wrapper {
+            margin : 2rem 0rem;
+            /** Start of container **/
+            &.container{
+                padding: $container-padding;
+                @include grid-container(5 , auto , 1.5rem);
+                &.two-col-layout{
+                    @include grid-container(2 , auto , 1.5rem);
+                }
+                  /** Start of add note **/
+                  .addNote{
+                    background-image: url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' stroke='%23A80000FF' stroke-width='3' stroke-dasharray='6%2c 9' stroke-dashoffset='0' stroke-linecap='square'/%3e%3c/svg%3e");
+                    @include flex-center();
```

Comment:
Include statements should be at the start of the list of property for a class, before adding any other property